SeleniumPracticalExercise/TestCases/AddEmployee.cs

```diff
@@ -1,31 +1,54 @@
 ﻿using NUnit.Framework;
+using SeleniumPracticalExercise.Common;
+using SeleniumPracticalExercise.PageObjects;
 using SeleniumPracticalExercise.TestCases.Common;
 
-namespace SeleniumPracticalExercise.TestCases
-{
-    class AddEmployee : BaseTestLocal
+class AddEmployee : BaseTestLocal
     {
         [Test]
         [Category("Add Employee")]
         public void AddEmployeeTest()
         {
+            string username = "Admin"; 
+            string password = "admin123"; 
+            string firstName = Utils.GenerateRandomString(6); 
+            string lastName = Utils.GenerateRandomString(8);
+
             // Steps to automate:
             // 1. Navigate to https://opensource-demo.orangehrmlive.com/web/index.php/auth/login
-            // 2. Log in using Username: Admin, Password: admin123
-            // 3. Click "PIM" in the left nav
+            Driver.Value.Navigate().GoToUrl("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
+
+            // 2. Log in using Username: Admin, Password: admin123 using LoginPage - Page Object Model
+            new LoginPage(Driver.Value).Login(username, password); 
+
+            // 3. Click "PIM" in the left nav using PimEmployee - Page Object Model
+            LeftNavPanel leftNavPanel = new LeftNavPanel(Driver.Value);
+            leftNavPanel.ClickPim();
+
             // 4. Click "+Add"
+            EmployeeInformationPage employeeInformationPage = new EmployeeInformationPage(Driver.Value);
+            employeeInformationPage.ClickAdd();
+
             // 5. Randomly generate a first name (6 characters) and last name (8 characters) and enter them into the form
-            // 6. Get the Employee Id for use later
+            // 6. Get the Employee Id for use later - using VerifyEmployee POM
             // 7. Click Save
+            AddEmployeePage addEmployeePage = new AddEmployeePage(Driver.Value);
+            string storeEmployeeID = addEmployeePage.GetIdNum();
+            addEmployeePage.SetEnterNames(firstName, lastName);      
+
             // 8. Click "PIM" in the left nav
+            leftNavPanel.ClickPim(); 
+
             // 8. Search for the employee you just created by Employee Id
+            employeeInformationPage.SetEmployeeIdField(storeEmployeeID);
+
             // 9. In the employee search results, use NUnit asserts to validate that Id, First Name, and Last Name are correct
-            //
-            // NOTE:
-            // - Use the provided WebDriver methods in BasePageLocal.cs
-            // - Create page objects as needed
-            // - Document all methods using XML documentation, https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/
+            string verifyIdResults = employeeInformationPage.EmployeeIdSearchResults();
+            string verifyFirstNameResults = employeeInformationPage.FirstNameSearchResults();
```

Comment:
Remember we talked about naming variables for asserts as "actual" or "expected"?

Reply:
I 100% did do this. Don't know why it wasn't showing but it renamed them again.

  string actualIdResults = employeeInformationPage.GetEmployeeIdSearchResults();
  string actualFirstNameResults = employeeInformationPage.GetFirstNameSearchResults();
  string actualLastNameResults = employeeInformationPage.GetLastNameSearchResults();